src/cmd/key/pass.sh

```diff
@@ -17,7 +17,11 @@ Try first:  $(basename $0) key join
       and:  $(basename $0) key split
 "
 
-    gpg --batch --no-tty --passwd $GPG_KEY
+    call_gpg key/pass.py $GPG_KEY
+
+    local err=$?
+    # TODO: use fail here after correcting test24. 
+    [[ $err == 0 ]] || echo "Error changing passphrase for $GPG_KEY" 2>&1
```

Comment:
I have done it in a similar way so that instead of using fail. Just to keep things same.